src/common/BrowserStorage.ts

```diff
@@ -387,6 +391,18 @@ class _BrowserStorage {
 			console.log('Upgrading to v10...');
 		}
 
+		if (version < 11 && this.currentVersion >= 11) {
+			console.log('Upgrading to v11...');
+
+			const { options } = await this.get('options');
+
+			if (options?.services && 'crunchyroll-beta' in options.services) {
+				delete options.services['crunchyroll-beta'];
```

Comment:
We should copy it to `options.services['crunchyroll']` before deleting it, no? Otherwise the user options for Crunchyroll will be lost (they'll have to re-enable it).

Reply:
I guess this ties back to https://github.com/trakt-tools/universal-trakt-scrobbler/pull/216#issuecomment-1356352686, so the users **should** re-enable it... In that case, disregard this and the comment below.

Still, it would be nice to warn people who have Crunchyroll enabled that they'd have to re-enable it, but that would be too much effort. I'm approving it.